src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Packaging/PluginInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Performance.Toolkit.PluginManager.Core.Discovery;
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Packaging
+{
+    // For local plugin, generate on installation
+    // For remote plugin, genereate on uploading to host.
+    public class PluginInfo : IPluginInfo
```

Comment:
I think `IPluginInfo` should be removed as an interface and this should just be a sealed class.